### ISSUE TITLE
Hyperpixel 4 touchscreen not responding to touch

### ISSUE THREAD

**Issue by spw1502** [pimoroni/hyperpixel4#123]
I have fitted a Hyperpixel 4 touch rectangle on Pi4 and installed software using the one line install script. Display works brilliantly except no touch response.
Re-installed software from github clone.
Still no touch operation.
This seems to be a perennial problem.
Have trawled around and can't find anything obvious else to try.
Am I missing something, this seems to have been an ongoing issue for at least a couple of years, surely it shouldn't be this difficult.

**Comment by tubbybtch** [comment]
Having the same issue...lack of response *troubling*

**Comment by Choirin** [comment]
I'm also having the same problem with a recently acquired one.
After some debugging, I found a resistor might be missing on my device.
(Pad on the right side, below the connector in the picture)
This resistor connects INT signal to raspberry pi.
I mount zero ohm resistor, and the touch function works fine.
![hyperpixel_int_res](https://user-images.githubusercontent.com/2378498/105979278-d5226100-60d6-11eb-9299-b73566cb54a1.jpg)

After the fix, I searched in google and found a picture which shows the resistor (4.7 kOhm) mounted.
https://deninet.com/blog/2019/06/30/handheld-raspberry-pi-part-1-if-first-you-dont-succeed

I'm guessing this issue may happen only on my device, but just in case, I'd like to share.
